packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -46,11 +48,25 @@ export const App = ({
               <LegalPrivacy path="/legal/privacy/*" />
               <LegalPrivacy path="/:locale/legal/privacy/*" />
 
-             <ResetPassword path='/reset_password/*' />
-             <ConfirmResetPassword path='/confirm_reset_password/*' />
-             <CompleteResetPassword path='/complete_reset_password/*' />
-             <ResetPasswordConfirmed path='/reset_password_verified/*' />
-             <ResetPasswordWithRecoveryKeyVerified path='/reset_password_with_recovery_key_verified/*' />
+              <ResetPassword path="/reset_password/*" />
+              <ConfirmResetPassword path="/confirm_reset_password/*" />
+              <CompleteResetPassword path="/complete_reset_password/*" />
+              {/* Pages using the Ready view need to be accessible to logged out viewers,
+               * but need to be able to check if the user is logged in or logged out,
+               * so they are wrapped in this component.
+               */}
+              <PageWithLoggedInStatusState
+                Page={ResetPasswordConfirmed}
+                path="/reset_password_verified/*"
+              />
```

Comment:
Sweet! I'd spotted earlier that this could just be a self-closing tag and you've already changed it. :) Nice to see how this looks with a few different pages using the `PageWithLoggedInStatusState` 👍